src/Components/Details.js

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
+import React, { Component } from 'react';
 import '../css/details.css';
 
-const Details = () => {
-  return (
-    <div>
-      <h1>Hello, this will be the details page for each Movie and TV Show</h1>
-    </div>
-  );
-}
+class Details extends Component {
+  constructor() {
+    super();
+    this.state = {message: "Hello, this will be the details page for each Movie and TV Show"}
```

Comment:
Could you please move the object to a new line, for readability purposes?

e.g.

```
this.state = {
    message: "Hello, this will be the details page for each Movie and TV Show"
}
```